setup.py

```diff
@@ -28,7 +28,7 @@
     "tensorflow-estimator==2.7.0",
     "tqdl==0.0.4",
     "psutil==5.7.0",
-    "transformers==4.5.1",
+    "transformers==4.19.2",
```

Comment:
this also needs changing in requirements.

Reply:
Have tests been run to verify that this change does not break anything else?

Reply:
added the change to requirements.txt.  i only ran the huggingface test suite, you think i should run everything (does transformers leak out of there to places i didn't notice)?

Reply:
Tokenizers is the main place as we use the HF tokenizers for roberta now.